bot.py

```diff
@@ -574,6 +577,18 @@ def main_color(self) -> int:
                 )
                 guild_age = self.config.remove("guild_age")
 
+        if not isinstance(thread_cooldown, isodate.Duration):
+            try:
+                thread_cooldown = isodate.parse_duration(thread_cooldown)
+            except isodate.ISO8601Error:
+                logger.warning(
+                    "The account age limit needs to be a "
```

Comment:
"The account age limit", copy-paste problem.

Reply:
Exposed 👀